lib/rwj_reporter.rb

```diff
@@ -6,14 +6,15 @@ def self.print_reports_for_dates(start_date_str, end_date_str, log_file_dir=nil,
   end
 
   def initialize(log_file_dir=nil)
-    @log_file_dir = if log_file_dir
-      log_file_dir
-    else
-      get_log_file_dir_from_settings
-    end
+    @log_file_dir =
+      if log_file_dir
+        log_file_dir
+      else
+        log_file_dir_from_settings
+      end
   end
 
-  def self.get_log_file_dir_from_settings
+  def self.log_file_dir_from_settings
```

Comment:
i don't know why this is the only `get_` method that rubocop complained about, and why i thought of this as an "accessor" method but not the others.